### DISCUSSION TITLE
New-HTMLTableStyle -Type Table -FontSize 25 not applying to tables

### DISCUSSION THREAD

**Discussion by HALnotSAFE** [EvotecIT/PSWriteHTML#428] [Q&A] [answered]
Hello, 

I am having an issue where I can't get the New-HTMLTableStyle to work on any of my tables. I have it before my New-HTML line (I believe this is the right spot) but my tables won't update their size. Am I missing something? 
```
New-HTMLTableStyle -Type Table -FontSize 25
    # Creates the HTML file
    New-HTML -Online -TitleText "Active Directory Dashboard" -FilePath "C:\users\public\desktop\ADDashboard.html" -AutoRefresh "90" {
```
I can't share much more of the code as it is for my work, but I would like to get the fontsize working to make it easier to view our tables on our monitor.

**Comment by PrzemyslawKlys** [comment]
New-htmltablestyle has to be within new-html, before the table as far as I remember.

**Reply by HALnotSAFE** [reply]
Still having issues with getting the New-HTMLStyle to apply even with it within the New-HTML. Here is how I have it formatted. I tried deleting the HTML to let it rebuild but no dice. 

New-HTML -Online -TitleText "Active Directory Dashboard" -FilePath "C:\users\public\desktop\ADDashboard.html" -AutoRefresh "90" {
        New-HTMLTableStyle -FontSize 25
        New-HTMLTab -Name "Active Directory Information" -TextSize "25px" {
            New-HTMLSection -Invisible {
                New-HTMLSection -HeaderText "Forest Information" -HeaderTextSize "20px" -Width 400px {
                    New-HTMLPanel -Margin "10px" {
                        "<p>Forest : $forestName</p>" 
                        "<p>Forest Functional Level : $($forest.ForestMode)</p>"
                        "<p>Domains :</p>"
                        "<ul>"
                        foreach($domain in $domains){
                            "<li>$($domain)</li>"
                        }
                        "</ul>"
                        "<p>Root Domain : $($forest.RootDomain)</p>"
                        "<p>Global Catalogs :</p>"
                        "<ul>"
                        foreach($catalog in $globalcatalogs){
                            "<li>$($catalog)</li>"
                        }
                        "</ul>"
                    }
                }
                New-HTMLSection -HeaderText "Domain Controllers" -HeaderTextSize "20px" {
                    New-HTMLPanel {
                        New-HTMLTable -DataTable $allDomainControllersTable -HideFooter -HideButtons -Simplify
                    }
                }
                New-HTMLSection -HeaderText "Enabled Vendor AD Accounts" -HeaderTextSize "20px" -Width 400px {
                    New-HTMLPanel{
                        New-HTMLTable -DataTable $allDomainInfo.vendoradaccounts -HideFooter -HideButtons -Simplify
                    }
                }
            }
            New-HTMLSection -Invisible {
                New-HTMLSection -HeaderText "Locked AD Accounts" -HeaderTextSize "20px" -Width 400px {
                    New-HTMLPanel {
                        New-HTMLTable -DataTable $allDomainInfo.lockedusers -HideFooter -HideButtons -Simplify
                    }
                }
                New-HTMLSection -HeaderText "AD Accounts that Expired One Month Ago" -HeaderTextSize "20px" {
                    New-HTMLPanel {
                        New-HTMLTable -DataTable $allDomainInfo.ExpiredUsersFirstHalf -HideFooter -HideButtons -AutoSize -Simplify
                    }
                    New-HTMLPanel {
                        New-HTMLTable -DataTable $allDomainInfo.ExpiredUsersSecondHalf -HideFooter -HideButtons -AutoSize -Simplify
                    }
                }
            }
        }
    }

**Reply by PrzemyslawKlys** [reply]
You can try to put it within new-htmltable. See if that works. Or find it in examples. Its there for sure. I am just not at work this week.

**Comment by HALnotSAFE** [comment, marked as answer]
Having simplify on the tables caused formatting to not work. Issue is resolved.
